docs/technical_reference/flowsheets/crystallization.rst

```diff
@@ -0,0 +1,81 @@
+NaCl Crystallization
+===============================
+
+Introduction
+------------
+
+Crystallization is the precipitation and extraction of the crystals from a mother liquor based on solute supersaturation. Crystallization is a potential solution to desalination's environmental brine management problem; brine crystallization systems have been shown to achieve zero-liquid discharge and salt recovery.
+
+The objective of the mathematical model is to simulate crystallization from a NaCl mother liquor. The example is based on (but does not replicate exactly) Example 11.4 in `Tavare, N.S. (1995). <https://link.springer.com/chapter/10.1007/978-1-4899-0233-7_11>`_ 
+
+Implementation
+--------------
+
+The modeled crystallization process is illustrated by Figure 1. Thermal energy for the evaporative crystallizer is added in the suspension heater, with the water vapor product sent off to a condenser (not modeled here). 
+
+The flowsheet relies on the following key assumptions:
+
+   * supports steady-state only.
+   * a property package supporting all three phases (e.g., the crystallizer property package) is provided for the crystallizer model.
+
+.. figure:: ../../_static/flowsheets/crystallizer.jpg
+    :width: 500
+    :align: center
+
+    Figure 1. Crystallizer flowsheet
+
+Documentation for the crystallizer unit model can be found below. 
+    * `Crystallizer <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/crystallizer_0D.html>`_
```

Comment:
One more thing I missed in my review - you should also provide a link here to the crystallization property package. Although it looks like there's no documentation for this yet... I think it'd be nice to have this for the 1.0 release, so maybe it's best to address this in a separate PR that also adds the prop pack documentation.